components/NavDrawer/NavDrawer.tsx

```diff
@@ -1,9 +1,7 @@
 'use client';
-
```

Comment:
w: Try to remove these white spaces and extra lines when you commit. It will look like your PR made changes to this file when we look at the history.

Reply:
👌 Will keep that in mind for future PR's! Good point!